commit 0bc7d520844ee4da54b2bf5ce4987c61b2732d90 (refs/changes/12/2012/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-05-29T13:56:39-07:00 (2 years, 4 months ago)
    
    IMGAPI-627 switch standalone IMGAPI to using ECDSA keys (latest smtools for DATASET-1269 fix for standalone IMGAPI)
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/targets.json.in b/targets.json.in
index 66c6015..6df3b2f 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -651,6 +651,7 @@ cat <<EOF
         "dateutils-0.3.1nb1",
         "haproxy-1.6.2",
         "openssl-1.0.2j",
+        "smtools-20160926",
         "stud-0.3p53nb5",
         "the_silver_searcher-0.31.0",
         "xz-5.2.2"
